coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/misc/KnownCase.java

```diff
@@ -7,7 +7,7 @@
  *
  * SPDX-License-Identifier: MPL-2.0
  */
-package org.coralibre.android.sdk.internal.database.models;
+package org.coralibre.android.sdk.internal.misc;
```

Comment:
same here.

Reply:
Let's create a new issue for cleaning these parts after the PR is done. Cleaning of these classes has not much to do with the database stuff.